components/rtttl.rst

```diff
@@ -0,0 +1,163 @@
+Rtttl Buzzer
+============
+
+.. seo::
+    :description: Instructions for setting up a buzzer to play tones and rtttl songs with ESPHome.
+    :image: crosshair-gps.png
+
+The ``rtttl``, component allows you to easily connect a passive piezo buzzer to your microcontroller
+and play monophonic songs. It accepts the Ring Tone Text Transfer Language, rtttl format (`Wikipedia
+<https://en.wikipedia.org/wiki/Ring_Tone_Transfer_Language>`__) which allows to store simple melodies.
+
+.. figure:: images/buzzer.jpg
+    :align: center
+    :width: 50.0%
+
+    Buzzer Module
+
+Overview
+--------
+
+It's important that your buzzer is a **passive** one, if it beeps when you feed it with 3.3V then it is not
+a passive one and this library will not work properly.
+
+The tone generator needs a PWM capable output to work with, currently only the
+:doc:`ESP8266 Software PWM Output<output/esp8266_pwm>` and
+:doc:`ESP32 LEDC Output <output/ledc>` are supported, i.e. outputs from port expanders boards like the
+:doc:`PCA9685 <output/pca9685>` are not supported for this component.
```

Comment:
PCA9685 is not supported atm right? I didn't see any `update_frequency` method for it (and it would be difficult to add, as frequency is shared across all channels)

Reply:
Saw a notification somewhere about this (can't find it anymore).

Yes I would just remove PCA9685 from this list.